pkg/workloads/console/runner/runner.go

```diff
@@ -56,7 +56,7 @@ type Options struct {
 	// expect a TTY to not be present?
 	// However with these types of consoles it will not be possible to send input to them
```

Comment:
I guess we lose this comment if we merge this! Maybe add one about why we have this flag.